recipes/rcat/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "RCAT" %}
+{% set version = "v1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jplind79/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c539ba7d0030119e47a0376859a3e670976a1aa4a6061f26480826dd4a113545
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
```

It looks like this could be noarch python?
```suggestion
  noarch: python
```

---

I'd go ahead and put this back to `noarch: python`. We can ignore the windows build failure, as noarch python packages are only built on Linux anyways. It's a good change to make since noarch python packages involve much less build overhead and are pretty much automatically available when new Python versions (e.g. 3.9) are released.